src/source/Stun/Stun.c

```diff
@@ -327,7 +327,7 @@ STATUS serializeStunPacket(PStunPacket pStunPacket, PBYTE password, UINT32 passw
 
             stunResult = StunSerializer_GetFingerprintBuffer(&stunContext, &pFingerprintBuffer, &buffereLength);
             CHK(stunResult == STUN_RESULT_OK, convertStunErrorCode(stunResult));
-            crc32 = COMPUTE_CRC32(pFingerprintBuffer, (UINT32) buffereLength) ^ STUN_FINGERPRINT_ATTRIBUTE_XOR_VALUE;
```

Comment:
Could you check if it's ok to remove `STUN_FINGERPRINT_ATTRIBUTE_XOR_VALUE` definition?

Reply:
Updated